trex-operator/Makefile

```diff
@@ -67,8 +67,16 @@ deploy: kustomize
 undeploy: kustomize
 	$(KUSTOMIZE) build config/default | ${CLUSTER_CLI} delete -f -
 
+# Ensure proper images for trex-container-app and trex-container-server
```

Comment:
Nice! It is good to see this method working. We will need at some point to change the way the T-REX operator is retrieving the digest it uses.

Reply:
100% agree